errors_validation.go

```diff
@@ -0,0 +1,47 @@
+package jsonschema
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type validationErrorContext interface {
+	addContext(instancePtr, schemaPtr string)
+	finishInstanceContext()
+}
+
+// ValidationRequiredError is used when a
+type ValidationRequiredError struct {
+	MissingPropertyPtrs []string
```

Comment:
lengthy name. required keyword works only for properties. so including `Property` in name feels like stutter. May be `MissingPtrs` or just `Missing` would be concise.